src/templates/layouts/storyListing/index.tsx

```diff
@@ -51,44 +51,46 @@ export function StoryListing({
     )
 
   return (
-    <div key={id} className="usa-grid usa-grid-full">
-      {/* Widget coming from vets-website */}
-      <nav
-        data-template="navigation/facility_sidebar_nav"
-        aria-label="secondary"
-        data-widget-type="side-nav"
-      ></nav>
+    <div id="content" className="interior">
```

Comment:
It feels like we can move this to a more common location so it's not needed in every template.

Reply:
Agreed, moved it to the slug component